src/screens/Profile.js

```diff
@@ -116,4 +142,9 @@ function AnimatedBackground({ height }) {
   )
 }
 
+function useQuery() {
```

Comment:
What about renaming it to useSearchParams? i needed to take a look to the code to understand what was that useQuery doing

Reply:
Yes sure!